emails/migrations/0028_copy_subdomain_to_registeredsubdomain.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.24 on 2021-10-15 05:22
+
+from django.db import migrations, models
+
+from emails.models import hash_subdomain
+
+def copy_already_registered_subdomains_from_profile(apps, schema_editor):
+    Profile = apps.get_model('emails', 'Profile')
+    RegisteredSubdomain = apps.get_model('emails', 'RegisteredSubdomain')
+    for profile in Profile.objects.exclude(subdomain=None):
+        RegisteredSubdomain.objects.create(subdomain_hash=hash_subdomain(profile.subdomain))
```

Comment:
Our `0025` migration did `Profile.objects.all()` which triggered out-of-memory error. But looks like this `exclude(subdomain=None)` should make this queryset ONLY get the Profiles of the few users who have gotten a subdomain via fox-fooding. So it should be 🆗